src/validators/with_default.rs

```diff
@@ -124,7 +143,13 @@ impl Validator for WithDefaultValidator {
         recursion_guard: &'s mut RecursionGuard,
     ) -> ValResult<'data, Option<PyObject>> {
         match self.default.default_value(py)? {
-            Some(dft) => {
+            Some(stored_dft) => {
+                let dft: Py<PyAny> = if self.copy_default {
+                    let deepcopy_func = py.import("copy")?.getattr("deepcopy")?;
```

Comment:
this should be stored behind a [`GilOnceCell`](https://docs.rs/pyo3/latest/pyo3/sync/struct.GILOnceCell.html), see `TIME_LOCALTIME` for an example.